libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -183,17 +184,25 @@ private QnAMakerOptions HydrateOptions(QnAMakerOptions queryOptions)
         private async Task<QueryResults> QueryQnaServiceAsync(Activity messageActivity, QnAMakerOptions options)
         {
             var requestUrl = $"{_endpoint.Host}/knowledgebases/{_endpoint.KnowledgeBaseId}/generateanswer";
+            AnswerSpanRequest answerSpanRequest = null;
+            if (options.EnablePreciseAnswer)
+            {
+                answerSpanRequest = new AnswerSpanRequest();
+                answerSpanRequest.Enable = options.EnablePreciseAnswer;
```

Comment:
You can use the object initializer here.

Reply:
Implemented the suggestion

Reply:
Updated. Please review if my understanding is correct.

---
In reply to: [433512499](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433512499) [](ancestors = 433512499)